net/net.c

```diff
@@ -5676,8 +5676,6 @@ static void *accept_thread(void *arg)
             last_stat_dump_time = pollend;
         }
 
-        netinfo_ptr->num_accepts++;
```

Comment:
We've already incremented it just [a few lines above](https://github.com/bloomberg/comdb2/pull/2293/files#diff-158d650f8c314f5cf37ed472eed53b5bL5670).